demisto_sdk/commands/common/tools.py

```diff
@@ -2214,3 +2214,46 @@ def get_pack_dir(path):
         if parts[index] == 'Packs':
             return parts[:index + 2]
     return []
+
+
+def get_scripts_and_commands_from_yml_data(data, file_type):
+    """Get the used scripts and commands from the yml data
+
+    Args:
+        data: The yml data as extracted with get_yaml
+        file_type: The FileType of the data provided.
+
+    Return (list of found command names, list of found script names)
+    """
+    commands = []
+    scripts = []
+    if file_type == FileType.TEST_PLAYBOOK or file_type == FileType.PLAYBOOK:
+        tasks = data.get('tasks')
+        for task_num in tasks.keys():
+            task = tasks[task_num]
+            inner_task = task.get('task')
+            task_type = task.get('type')
+            if inner_task and task_type == 'regular' or task_type == 'playbook':
+                if inner_task.get('iscommand'):
+                    commands.append(inner_task.get('script'))
```

Comment:
Lets handle the splitting logic here. We will split command to dict of brand and command name. It will really simplify the code from other places and seems like a logic that should be common (Everyone calling this function will probably want the data splitted anyway)
```suggestion
                    task_script_parts = inner_task.get('script')
                    if len(task_script_parts) == 2:
                        commands.append({'brand': task_script_parts[0], 'command': task_script_parts[1]})
```